### DISCUSSION TITLE
Why use both LanceDB and Parquet?

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [microsoft/graphrag#787] [Q&A]
I just noticed that graphrag creates a directory `lancedb` outside of `ragtest`. [LanceDB](https://github.com/lancedb/lance) is a columnar data format for ML and pitches itself as an alternative to Parquet.

Is there a reason why GraphRAG needs both of these?

**Comment by natoverse** [comment]
Parquet is used for general storage, lancedb is used for vector search, so only the embeddings are loaded there.
